### PR TITLE
Implement board converter

### PR DESCRIPTION
**Pull request by alena-bartosh**

### CODE REVIEW

File: src/board_converter.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class BoardConverter:
+    @staticmethod
+    def to_string(board: Board) -> str:
+        result = ''
+
+        for line in board.get_cells():
+            for cell in line:
+                result += '1 ' if cell else '0 '
+
+            result = result.rstrip(' ')
```
Comment (s373r) [inline src/board_converter.py:14]: looks like a bug

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/board_converter.py (hunk 1 above)
Comment (s373r) [reply src/board_converter.py:14]: here we override the previous value of `result`

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/board_converter.py (hunk 1 above)
Comment (alena-bartosh) [reply src/board_converter.py:14]: But we want to do this!
```
rstrip() вернет копию строки str с удаленными символами chars в конце строки.
```

**Review by s373r — APPROVED** [review]

**Review by Serg-Bartosh — APPROVED** [review]
@Serg-Bartosh , you are so goon , well done :crab:

**Pull request merged**